monolithic/pkg/assembler.go

```diff
@@ -254,31 +257,44 @@ func RunMonolithicLamassuPKI(conf MonolithicConfig) (int, error) {
 		buildReverseProxyHandler(engine, "VA", "/api/va/", vaPort)
 		buildReverseProxyHandler(engine, "Alerts", "/api/alerts/", alertsPort)
 
-		listener, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", conf.GatewayPort))
+		listenerHttps, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", conf.GatewayPortHttps))
+		if err != nil {
+			log.Fatalf("could not get Gateway net Listener: %s", err)
+		}
+
+		listenerHttp, err := net.Listen("tcp", fmt.Sprintf("0.0.0.0:%d", conf.GatewayPortHttp))
 		if err != nil {
 			log.Fatalf("could not get Gateway net Listener: %s", err)
 		}
 
-		usedPort := listener.Addr().(*net.TCPAddr).Port
+		usedHttpsPort := listenerHttps.Addr().(*net.TCPAddr).Port
+		usedHttpPort := listenerHttps.Addr().(*net.TCPAddr).Port
 
 		go func() {
-			// log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", conf.GatewayPort), engine))
-
-			server := http.Server{
+			serverHttps := http.Server{
 				Handler: engine,
-				Addr:    fmt.Sprintf(":%d", conf.GatewayPort),
+				Addr:    fmt.Sprintf(":%d", conf.GatewayPortHttps),
```

Comment:
It is recommended to set a ReadHeaderTimeout in the http.Server config to prevent Slowloris Attacks. 
Please set a timeout here